source/our_tools/flows.rst

```diff
@@ -0,0 +1,252 @@
+.. _ref_flow:
+
+Flows
+=====
+
+.. role:: bash(code)
+   :language: bash
+
+Here we'll look at how to move from the raw diffusion imaging data to the tractography metrics, whether for global visualization or at the scale of the invididual, using flows created by our laboratory.
+
+Requirements
+************
+
+* For this test, there is a necessity that you have a Nextflow version between 19.04.2 and 21.12.1, the various git flows cloned (tractoflow, rbx_flow, tractometry_flow) and have a container installed.
+* For Nextflow and tractoflow you can check the `installation guide <https://tractoflow-documentation.readthedocs.io/en/latest/installation/install.html>`_ from the tractoflow documentation. Otherwise, the installation of Nextflow is also presented `here <https://scil-documentation.readthedocs.io/en/latest/intro_to/explore_nextflow.html#installation>`_.
```

Comment:
For the moment I don't know what will happen to the tractflow section, but it is normally planned to delete it.

Reply:
I agree. But there were some important information about how to run it on beluga, using sbatch, etc. But it is probably not up to date. I think this should be managed in this PR, or soon. @frheault , @arnaudbore , any insight?

Reply:
I think both should be separate. Launching a series of flows on your computer or on ComputeCanada is basically the same task (the sequence, chaining portion only).

The ComputeCanada page should focus on file organisation (good practices), setting up the modules, rsync and the basics commands (quota, salloc, sbatch).

The page about flows on Belgua should focus on parameters that must be added, warning on quota, resources estimation and QC good practices.

I think that trying to do one page to do it all would be more confusing (it would triple in length the current PR)

This PR is part one, then maybe @anroy1 could do the 'flows on Beluga' (he just did one project with Arnaud)
Then I think the ComputeCanada page (DEALING WITH HEAVY DATA) could simply bit expanded a bit.